modules/chameleon/src/main/java/org/clueminer/chameleon/PairMerger.java

```diff
@@ -173,8 +173,8 @@ protected void addIntoTree(int clusterIndex1, int clusterIndex2) {
         if (sim > 10) {
             sim = 10;
         }
-        if (sim < 0.01) {
-            sim = 0.01;
+        if (sim < 0.005) {
```

Comment:
I am trying to find a method to set this value dynamically, this is work in progress.